javascript/javascript1/week1/house-price.js

```diff
@@ -0,0 +1,19 @@
+// peter house measures
+let wide = 8;
+let deep= 10;
+let high = 10;
+let gardenSizeInM2 = 100;
+let volumeInMeters = wide * deep * high
+let housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+console.log(housePrice)
+
+// julia house measures
+wide = 5
+deep = 11
+high = 8
+gardenSizeInM2 = 70
+volumeInMeters = wide * deep * high
+housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+console.log(housePrice)
+
```

Comment:
Have you answered the question of whether Julia or Peter should buy the house?

Try and log the following for each person: 
```js
console.log(`Peter has to pay: {housePrice}`)
console.log(`The value of the house is : {houseValue}`)
console.log(`Peter should buy the house: {shouldBuyTheHouse}`)
```
Where `shouldBuyTheHouse` is a boolean value.